### PR TITLE
Upate Drush Import Doc

### LINKED ISSUE #7506: [CMS-125] Manually Migrate Sites to Pantheon Doc Update
Re: [Manually Migrate Sites to Pantheon](https://pantheon.io/docs/migrate-manual)

Priority: Medium

## Issue Description:

We've added the ability to use `drush archive-dump` and Terminus to import sites to Pantheon, and the steps need to be documented here.

### Setup
* Ensure that you have Terminus 3 installed on your local system: https://pantheon.io/docs/terminus/install
* Install Terminus Conversion Tools plugin: https://github.com/pantheon-systems/terminus-conversion-tools-plugin#installation
* Install Drush 11.1.1 or higher: https://www.drush.org/latest/install/

### Dump Site
* In your local site directory, run `drush archive:dump` to create an `archive.tar.gz` file which is a backup of your site

### Import 
* To create the site on Pantheon, run `terminus conversion:import-site site-machine-name /path/to/archive.tar.gz --site-label="Site Label"`

### PR DESCRIPTION
**Pull request by whitneymeredith**
Closes #7506 

## Summary

**[Importing Drush Site Archives with Terminus](https://pantheon.io/docs/drush-import)** - Updated Drush import steps to include steps for D9 Composer-based sites.

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-7514](https://pr-7514--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `45` | `51` | `6` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `98` | `98` | `0` |

Lighthouse Audit Reports:

- [`pr-7514` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/39622/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/39622/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-7514--pantheon-docs.my.pantheonfrontend.website/docs)

**Comment by whitneymeredith** [comment]
[Preview](https://pr-7514--pantheon-docs.my.pantheonfrontend.website/docs/drush-import)

File: source/content/drush-import.md [hunk 1]
```diff
@@ -1,37 +1,46 @@
...
 
-There are a few things you'll need in order to make this work:
+1. Create a Pantheon account with at least one free Dev site slot open. 
+
+   - A [Pantheon account is free](https://dashboard.pantheon.io/register), and if you need an extra Dev site to try this out, just ask and we'll be happy to grant you one.
 
-1. A Pantheon account with at least one free Dev site slot open. A [Pantheon account is free](https://dashboard.pantheon.io/register), and if you need an extra Dev site to try this out, just ask and we'll be happy to grant you one.
-2. A working local Drush installation that is up to date with 7.x or 8.x stable.
-3. Drush access to your existing Drupal site.
+1. Verify that you have Drupal 7 with Drush 8 if you have a non-Composer managed site *or* Drupal 9 with [Drush 11.1.1 or higher](https://www.drush.org/latest/install/) if you have a Composer-managed site.
```
Comment (greg-1-anderson) [inline source/content/drush-import.md:19]: I might phrase this differently.

"If you have a Drupal 7 site, verify that you are using Drush 8. Note that Composer-managed sites are not supported."

"If you have a Drupal 9 site, verify that you are using Drush 11. Note that **only** Composer-managed sites are supported."

Maybe we can point other, unsupported configurations back to the "manually migrate" pages?

File: source/content/drush-import.md [hunk 2]
```diff
@@ -1,37 +1,46 @@
...
+   - This creates a file called `drush-archive.tar.gz` that's available via the public internet. If you have the file locally, you can put it on Dropbox, S3, or any number of other places. The important thing is that you have a Drush archive that can be downloaded via a URL.
 
 ## Import Your Archive
 
-Importing a Drush site archive as we've prepared it above is easy.
+<TabList>
+
+<Tab title="Drupal 7 non-Composer" id="d7" active={true}>
+
+1. Set up Terminus, the Pantheon CLI, if you haven't already using Composer as described in the [installation instructions on GitHub](https://github.com/pantheon-systems/cli/wiki/installation).
```
Comment (greg-1-anderson) [inline source/content/drush-import.md:43]: The Terminus 3 install instructions in the other tab should be fine for this use-case as well. Maybe we should pull these up, outside the tab group.

File: source/content/drush-import.md [hunk 3]
```diff
@@ -53,8 +62,33 @@ Importing a Drush site archive as we've prepared it above is easy.
...
+
+</Tab>
+
+<Tab title="Drupal 9 Composer-managed" id="d9">
+
+1. Install [Terminus 3](/terminus/terminus-3-0) if you have a Composer-managed Drupal 9 site with Drush 11.1.1.
+
+1. Install the [Terminus Conversion Tools](https://github.com/pantheon-systems/terminus-conversion-tools-plugin#installation) plugin. 
+
+1. Run the command below to create the site on Pantheon. Change the `site-machine-name` to your machine's name.
```
Comment (greg-1-anderson) [inline source/content/drush-import.md:74]: > Change the `site-machine-name` to your machine's name.

This sounds like there is an existing site. Maybe "Change the `site-machine-name` to a unique name that you would like to use for your site."?

**Review by greg-1-anderson — COMMENTED** [review]
Just a few comments; overall, looks good.

**Review by michellecolon-pantheon — APPROVED** [review]
LGTM

**Pull request merged**